### PR TITLE
Fix login page validations

### PR DESCRIPTION
**Pull request by adeelkhan**
1) Removing focusout calls.
2) Adds email validation for less than 3 characters.
Screen Shot 2021-02-16 at 3 30 09 AM

VAN-362

### CODE REVIEW

File: src/login/tests/LoginPage.test.jsx [hunk 1]
```diff
@@ -161,13 +161,14 @@ describe('LoginPage', () => {
     );
   });
 
-  it('updates the error state for invalid password', () => {
```
Comment (zainab-amir) [inline src/login/tests/LoginPage.test.jsx:164]: any reason why we removed the invalid password test? Can we accommodate it a new test or in the test above?
`updates the error state for invalid email; no input` => `updates the error state for empty form`

File: src/login/tests/LoginPage.test.jsx [hunk 2]
```diff
@@ -176,26 +177,25 @@ describe('LoginPage', () => {
     );
   });
 
-  it('should update the error message on focus out', () => {
-    const errorState = { email: 'Please enter your Email.', password: 'Please enter your Password.' };
+  it('updates the error state for invalid email; format validation', () => {
+    const errorState = { email: '', password: '' };
```
Comment (zainab-amir) [inline src/login/tests/LoginPage.test.jsx:181]: we can add the error message here 
```suggestion
    const errorState = { email: 'The email address you\'ve provided isn\'t formatted correctly.', password: '' };
```

and remove it from here https://github.com/edx/frontend-app-authn/blob/a25695e57645ccf40e49df04247af35f392bacb6/src/login/tests/LoginPage.test.jsx#L186 and https://github.com/edx/frontend-app-authn/blob/a25695e57645ccf40e49df04247af35f392bacb6/src/login/tests/LoginPage.test.jsx#L192

File: src/login/tests/LoginPage.test.jsx (hunk 1 above)
Comment (adeelkhan) [reply src/login/tests/LoginPage.test.jsx:164]: No intent of removing any tests. I guess somehow editor did that for me and i failed to recognize it on Github on a quick glance. I have again added the missing password test and changed the tests name to be more expressive.

**Comment by adeelkhan** [comment]
@zainab-amir Please review again.

**Review by zainab-amir — APPROVED** [review]

**Pull request merged**